packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -10,7 +10,7 @@ import { makeStorageNodeChild } from '@agoric/vats/src/lib-chainStorage.js';
 import { makeRatio } from '@agoric/zoe/src/contractSupport/index.js';
 import { E, Far } from '@endo/far';
 import { Stable, Stake } from '@agoric/vats/src/tokens.js';
-import { deeplyFulfilled } from '@endo/marshal';
+import { deeplyFulfilledObject } from '@agoric/zoe/src/contractSupport/zoeHelpers.js';
```

Comment:
Can we put it somewhere we don't need to deeply import to get it?

Reply:
Out of scope, imo. There are many `zoeHelpers.js` exports already and if we want to stop that there should be a separate PR. Note that until we can get [ESM subpath exports working](https://github.com/Agoric/agoric-sdk/pull/6001#discussion_r950704872) it'll mean putting all exports in one path, and I don't think these helpers belong at `@agoric/zoe`.

Reply:
Resolving